SwiftUI_DZ/SwiftUI_DZ/View/VPNView.swift

```diff
@@ -0,0 +1,215 @@
+                            receivedTextView
+                            sentTextView
+                        }
+                        .padding()
+                        Spacer()
+                        expireTextView
+                    }
+                    .animation(.default)
+                }
+            }
+        }
+    }
+    
+    // MARK: - Private Properties
+    @StateObject private var stopWatchManager = StopWatchManagerViewModel()
+    @State private var isToggleOn = false
+    @State private var onoff = false
+    
+    private let smallRadial = RadialGradient(
+        gradient: Gradient(colors: [.yellow, .red, .white, .green]),
+        center: .bottom,
+        startRadius: Constants.startRadius,
+        endRadius: Constants.endRadius)
+    
+    private var timeTextView: some View {
+        Circle()
+            .frame(width: Constants.timeTextCircleFrame,
+                   height: Constants.timeTextCircleFrame)
+            .foregroundColor(.yellow)
+            .overlay {
+                Text(stopWatchManager.timeText)
+                    .foregroundColor(.purple)
+                    .font(.title)
+                    .bold()
+            }
+    }
+    
+    private var timeCircle: some View {
+        ZStack {
+            withAnimation {
+                Circle()
+                    .trim(from: Constants.timeCircleTrimFrom,
+                          to: Constants.timeCircleTrimTo)
+                    .stroke(.yellow, style: StrokeStyle(lineWidth: Constants.strokelineWidth,
+                                                        lineCap: .round))
+                    .frame(width: Constants.timeCircleFrame,
+                           height: Constants.timeCircleFrame)
+            }
+            
+            withAnimation {
+                Circle()
+                    .trim(from: Constants.timeCircleTrimFrom, to: min(stopWatchManager.circleValue, Constants.oneCountDouble))
+                    .stroke(smallRadial, style: StrokeStyle(lineWidth: Constants.strokelineWidth,
+                                                            lineCap: .round))
+                    .frame(width: Constants.timeCircleFrame,
+                           height: Constants.timeCircleFrame)
+                    .rotationEffect(.degrees(Constants.timeCircleRotationEffect))
+            }
+        }
+    }
+    
+    private var CountryFlagImageView: some View {
+        Image(Constants.countryImageName)
+            .resizable()
+            .scaledToFill()
+            .frame(width: Constants.countryImageFrame,
+                   height: Constants.countryImageFrame)
+            .cornerRadius(Constants.countryImageCornerRadius)
+            .overlay(Circle()
+                .stroke(smallRadial, lineWidth: Constants.countryStrokeLineWidth)
+                .shadow(color: .white, radius: Constants.countryShadowRadius))
+            .blur(radius: isToggleOn ? Constants.cgZero : Constants.countryBlurRadius)
```

Comment:
```suggestion
            .blur(radius: 
```
логику во вью модель